### ISSUE TITLE
metabolite Identifiers are not in the COBRApy-parsed Mouse-GEM.xml

### ISSUE THREAD

**Issue by gmhhope** [SysBioChalmers/Mouse-GEM#13]
### Description of the issue:
Metabolite Identifiers are not in the COBRApy-parsed `Mouse-GEM.xml`. The same scenario are in `Rat-GEM.xml`, `Mouse-GEM`, `Rat-GEM`, `Worm-GEM` and `Zebrafish-GEM`(also see https://github.com/SysBioChalmers/Zebrafish-GEM/issues/5). In addition, `Fruitfly-GEM` cannot be parsed using COBRApy(see https://github.com/SysBioChalmers/Fruitfly-GEM/issues/5).

#### Expected feature/value/output:
Expect to have identifiers in `.xml` files synchronized with flatfiles `metabolite.tsv`. 

#### Current feature/value/output:
In humanGEM, everything is nicely showed up:
```py
model.metabolites[25].__dict__
```
```
{'_id': 'MAM00009r',
 'name': '(11Z,14Z)-eicosadienoyl-CoA',
 'notes': {},
 '_annotation': {'sbo': 'SBO:0000247',
  'kegg.compound': 'C16180',
  'chebi': 'CHEBI:76410',
  'lipidmaps': 'LMFA07050036',
  'vmhmetabolite': 'CE4843',
  'metanetx.chemical': ['MNXM167435', 'MNXM6496']},
 '_model': <Model HumanGEM at 0x7fcff72836a0>,
 '_reaction': {<Reaction MAR02901 at 0x7fcfec58fe50>,
  <Reaction MAR03002 at 0x7fcfeb423130>,
  <Reaction MAR03492 at 0x7fcffda16e80>,
  <Reaction MAR03681 at 0x7fcfeea65fd0>},
 'formula': 'C41H66N7O17P3S',
 'compartment': 'r',
 'charge': -4,
 '_bound': 0.0}
```

But in Mouse-GEM, Worm-GEM ...:
Same codes result in:
```
{'_id': 'MAM00012c',
 'name': '(11Z,14Z,17Z)-eicosatrienoyl-CoA',
 'notes': {},
 '_annotation': {'sbo': 'SBO:0000247'},
...,
 'formula': 'C41H64N7O17P3S',
 'compartment': 'c',
 'charge': -4,
 '_bound': 0.0}
```

**I hereby confirm that I have:**
- [ ] Tested my code with [all requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---user) for running the model
- [x] Done this analysis in the `master` branch of the repository
- [x] Checked that a similar issue does not exist already
- [ ] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/yeast-GEM) about the issue

**Comment by haowang-bioinfo** [comment]
@gmhhope thanks for reporting this. This bug should be fixed by applying `annoateGEM` to **Mouse-GEM** before exporting as `.xml`.

**Comment by gmhhope** [comment]
> @gmhhope thanks for reporting this. This bug should be fixed by applying `annoateGEM` to **Mouse-GEM** before exporting as `.xml`.

Dear Haowang,

Thanks for replying me! Could you elaborate on what exactly the function `annotateGEM` from and how to apply? And I was trying to read your `Mouse-GEM.xml` file instead of exporting it. So I got confused.

Thanks,
Minghao Gong

**Comment by haowang-bioinfo** [comment]
@gmhhope the [annoateGEM](https://github.com/SysBioChalmers/Human-GEM/blob/main/code/annotateGEM.m) function is from Human-GEM and usually applied while releasing new version. We'll incorporate this step to Mouse-GEM in future release.

**Comment by haowang-bioinfo** [comment]
this has been fixed in v1.3
